lib/adal-angular.js

```diff
@@ -89,6 +89,7 @@ if (typeof module !== 'undefined' && module.exports) {
                         }
 
                         if (requestInfo.requestType !== _adal.REQUEST_TYPE.LOGIN) {
+                            _adal._renewActive = false;
```

Comment:
Sorry I'm putting the comment in a wrong line - I think we should add a broadcast line in line 111 for errors, as the returned hash may contain the errro_description from AAD.:
if (requestInfo.parameters['access_token']) {
                                        _adal.callback(_adal._getItem(_adal.CONSTANTS.STORAGE.ERROR_DESCRIPTION), requestInfo.parameters['access_token']);
                                        return;
                                    } else if (requestInfo.parameters['id_token']) {
                                        _adal.callback(_adal._getItem(_adal.CONSTANTS.STORAGE.ERROR_DESCRIPTION), requestInfo.parameters['id_token']);
                                        return;
                                    } else {
                                        $rootScope.$broadcast('adal:loginFailure', _adal._getItem(_adal.CONSTANTS.STORAGE.ERROR_DESCRIPTION));
                                    }